packages/math-input/less/main.less

```diff
@@ -1,5 +1,5 @@
-@import "echo.less";
-@import "overrides.less";
-@import "popover.less";
+@import "./echo.less";
+@import "./overrides.less";
+@import "./popover.less";
 @import "../../../node_modules/mathquill/build/mathquill.css";
-@import "tabbar.less";
+@import "./tabbar.less";
```

Comment:
I assume this was necessary now that we're import main.less from perseus-renderer.less.

Reply:
I guess it wasn't? But I figured being explicit is nicer? idk.